src/infra/accounts/repo-tests/src/password_hash_repository_test_suite.rs

```diff
@@ -76,3 +96,72 @@ pub async fn test_store_couple_account_passwords(catalog: &Catalog) {
 }
 
 ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+pub async fn test_modify_password(catalog: &Catalog) {
```

Comment:
Please add a test with an attempt to set an empty password

Reply:
PS. The same applies to the password hash

Reply:
The value type you suggested (nutype-based) would handle those better.